index.js

```diff
@@ -26,14 +26,15 @@ function handleError (err) {
 }
 
 module.exports = function BubleLoader(source, inputSourceMap) {
+    var globalOptions = this.options.buble || {};
```

Comment:
I think fallback ( `|| {}` ) is redundant, since `Object.assign()` ignores `null`, `undefined` like: `Object.assign(a = {a: 1}, undefined, b = {b: 1})` eventually being just `{a: 1, b: 2}`